### PR TITLE
[FIX] UN-1068 Removing unused filetypes

### PR DESCRIPTION
**Pull request by harini-venkataraman**
## What
Removing unsupported filetypes for I/O Connectors from the list.

## Why
Redundant filetypes mentioned in the list

## How
Removing the list from the JSON Schema

## Database Migrations
Not applicable.

## Env Config
Not applicable.

## Screenshots
image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by nehabagdia — APPROVED** [review]

File: backend/workflow_manager/endpoint/static/src/file.json [hunk 1]
```diff
@@ -25,9 +25,8 @@
                 "enum": [
                     "PDF documents",
                     "Text documents",
-                    "Word documents",
-                    "Openoffice documents",
                     "Images"
+
```
Comment (nehabagdia) [inline backend/workflow_manager/endpoint/static/src/file.json:29]: Nit: Please remove this extra line.

**Review by tahierhussain — APPROVED** [review]
LGTM

**Review by chandrasekharan-zipstack — CHANGES_REQUESTED** [review]
@harini-venkataraman please check [this validation](https://github.com/Zipstack/unstract/blob/afa50665be69fcf0398c8d29086b3b38ac7b8249/backend/workflow_manager/endpoint/source.py#L112-L118) and remove the unused file types from this as well

**Comment by muhammad-ali-e** [comment]
@harini-venkataraman @chandrasekharan-zipstack @nehabagdia @tahierhussain 
 Isn't this validation based on various tools? Some tools support images, some support PDFs, and some both, right? So, why do we need to remove it from here?
 
 @hari-kuriakose

**Comment by harini-venkataraman** [comment]
> @harini-venkataraman @chandrasekharan-zipstack @nehabagdia @tahierhussain Isn't this validation based on various tools? Some tools support images, some support PDFs, and some both, right? So, why do we need to remove it from here?
> 
> @hari-kuriakose

@muhammad-ali-e Word Document & Open office document, which we dont currently support are listed and this PR removes those. We have removed only those file types from validations and constants. PDF, word and images are retained.

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM, do assess if we persist this data and if we might require migrations for it. I doubt if anyone might have set any of these options but its always good to be sure

**Comment by harini-venkataraman** [comment]
> LGTM, do assess if we persist this data and if we might require migrations for it. I doubt if anyone might have set any of these options but its always good to be sure

@chandrasekharan-zipstack Currently we don`t persist filetype in database. Validation is done in tools for allowed filetypes apart from the files changed here - [Ref.](https://github.com/Zipstack/unstract-sdk/blob/c46fdc075e5e752b659599b6d92b5e0367265c1b/src/unstract/sdk/tool/validator.py#L200) While a new tool is developed we must ensure that non supported types are not used.

**Pull request merged**